libs/ngx-mime/src/lib/viewer/viewer-header/viewer-header.component.ts

```diff
@@ -71,7 +71,7 @@ export class ViewerHeaderComponent implements OnInit, OnDestroy {
   isContentSearchEnabled = false;
   isFullscreenEnabled = false;
   isInFullscreen = false;
-  fullscreenLabel = this.intl.fullScreenLabel;
+  fullscreenLabel = '';
```

Comment:
Why is this removed?

Reply:
```this.intl``` gives NPE. Initalization moved to ngOnInit